rest-service/manager_rest/flask_utils.py

```diff
@@ -0,0 +1,80 @@
+#########
```

Comment:
Moved it here to avoid circular dependency (was previously located in `manager_rest/utils.py`)